source/Steamworks_gml/extensions/steamworks/steamworks_cpp/GMLSteam/steam_networking.cpp

```diff
@@ -140,16 +143,19 @@ CSteamID steam_net_packet_sender;
 /// Receives a packet, returns whether successful (retrieve data via steam_net_packet_).
 YYEXPORT void /*double*/ steam_net_packet_receive(RValue& Result, CInstance* selfinst, CInstance* otherinst, int argc, RValue* arg)//() 
 {
+	uint32 value32 = (argc >= 0) ? YYGetInt32(arg, 0) : 0;
```

Comment:
Same as before if you are checking for `YYGetInt32(.., 0)` you need to check if `argc > 0`